webclient/app/src/app/features/observationArea/ObservationAreaCard.jsx

```diff
@@ -45,11 +62,26 @@ function ObservationAreaCard(props) {
                 <Divider />
                 <CardActionArea onClick={openArea}>
                     {observationArea.image !== null ?
+                        <>
                         <CardMedia
                             component="img"
                             height="300"
-                            src={imageUrl}
-                        /> :
+                            src={imageUrl}>
+                        </CardMedia>
+                        <ImageListItemBar
```

Comment:
kannst du hier bitte die gesamte ImageListItemBar in die renderProcessing-Icon Methode auslagern? Das wird doch jetzt auch rot angezeigt, wenn das nicht enabled ist. CardMedia sollte zudem nicht doppelt drin sein.